.github/workflows/rust-ci.yml

```diff
@@ -3,22 +3,33 @@ on:
   workflow_call:
     inputs:
       rust_msrv:
-        description: The minimum support Rust version for your project, e.g. 1.71.0.
+        description: >
+          The minimum support Rust version to test the project with,
+          e.g. 1.71.0.
         required: true
         type: string
-      test_minimal_versions:
-        description: Test using minimal dependency versions from Cargo.toml.
+      test_minimal_version:
+        description: > 
+          If non-empty, test using minimal dependency versions from Cargo.toml
```

Comment:
This description fails to mention that the string should be a Rust version number, I think that would be helpful.